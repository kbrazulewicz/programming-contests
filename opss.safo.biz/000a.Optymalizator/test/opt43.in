10
500 -211960
499 -225 250 -461 -43 281 -444 234 -76 -135 -262 101 110 119 -326 -309 -406 108 -33 -271 -477 -133 116 -85 -15 -231 299 217 228 93 -391 -421 368 -288 -382 178 -155 -326 264 269 391 -498 -130 354 122 396 -455 -432 5 364 -203 380 -269 414 147 -432 35 -54 -362 115 -461 -253 195 -240 311 -335 -210 -344 -160 -446 -223 83 -444 -500 -63 -470 396 334 -402 253 198 248 486 -218 -486 133 202 401 -68 340 17 -29 -60 -288 -417 -249 -123 -127 260 -431 279 -463 -495 336 -463 -58 -282 -214 129 169 -109 -173 -83 -123 461 -69 -137 164 184 295 356 -299 118 148 265 202 400 -5 -73 -488 416 -293 -451 421 -105 -61 -136 113 225 493 134 -384 172 -449 346 486 334 -291 150 19 -496 358 -428 122 7 338 176 259 -167 -44 271 -251 -485 320 23 -90 -241 387 -125 -164 -268 -490 305 -244 -87 151 -258 -252 360 -256 -381 -284 -45 -309 338 462 -119 367 221 214 175 344 -184 190 164 339 -48 -224 -422 327 464 162 -311 269 418 103 -80 13 -297 132 -391 174 348 64 365 -461 26 99 -242 99 -187 433 -57 481 123 -40 -328 -425 -412 102 -246 -448 -236 -56 -326 35 -101 446 -100 454 79 9 128 -73 -426 345 318 452 -56 -424 404 -390 -491 199 -57 484 -341 116 -89 99 70 166 -496 -313 -38 -322 222 -287 -376 -26 -333 -445 483 295 335 -91 492 153 214 -63 -418 118 399 -409 -183 -158 -72 329 -190 339 280 381 357 284 -432 -181 -186 -358 32 291 468 -449 346 -197 -302 181 213 -309 -165 279 480 269 249 379 -288 -82 -279 140 -253 384 332 -120 -383 189 -336 37 -139 331 179 -255 122 499 -203 320 -198 -5 2 -485 186 -311 -354 166 458 395 397 -330 -186 -29 -337 413 -145 495 -207 -28 184 310 -491 -455 493 -460 -209 115 -109 440 -65 193 -65 -211 -439 -26 -22 -293 -8 288 -397 -110 459 -83 -287 -378 182 68 -31 476 -108 -347 -362 253 51 -369 -355 -306 -402 36 -366 33 -271 421 -325 -210 -105 153 350 388 294 453 130 -395 222 343 79 -96 263 -452 232 7 53 370 112 -396 -147 257 150 -49 145 -364 -163 -274 -442 12 17 -47 17 -133 193 -189 172 175 -84 -106 370 -153 -350 133 -105 383 -8 449 -395 -396 -95 -41 361 -444 -238 6 44 451 -415 102 -37 102 -92 -167 -179 -47 496 -155 -371 265 239 499 -388 -259 485 8 -376 -23 309 82 82 -286 -107 295 122 155 154 -333 107 239 -379 -78 193 29 -393 -134 483 -396 211 -36 
500 10221
302 315 185 -457 300 45 20 130 -146 -398 64 421 347 -141 43 2 -487 -438 -39 -396 -316 -264 297 -435 -157 -337 -100 299 374 364 -128 -472 180 -442 -428 332 103 444 314 458 -454 -122 379 -255 238 274 247 -397 -311 61 -292 -127 -351 -143 -62 -8 21 339 144 247 -445 -484 276 -413 -74 -300 -228 -470 -356 -414 -160 42 317 71 287 55 197 386 -490 386 -53 -282 111 96 428 -98 -59 -51 93 -63 48 148 305 -176 -412 -268 376 -140 114 -480 298 454 62 115 377 -151 22 74 -264 33 313 -465 -397 -76 484 -469 326 -75 332 -81 362 -119 -80 167 -443 8 -249 434 -280 365 454 -482 -329 17 486 -452 -282 8 475 -46 -459 288 342 -3 64 326 28 243 103 -139 -338 -35 -406 82 -16 -349 -58 -412 -415 162 453 -108 -467 -375 -239 -481 25 -21 27 352 -214 421 492 -372 -82 57 -194 446 -200 409 159 314 226 -395 249 62 -243 -309 150 194 -294 104 -414 -261 -419 199 -242 106 -321 285 -41 317 -442 -49 -203 -24 -140 103 275 160 364 -66 -25 90 40 -276 -348 -351 267 303 -305 473 259 134 -288 340 -167 322 298 364 108 -243 181 166 -439 -170 -5 -79 -215 -230 -66 -351 -444 409 -409 96 -15 96 -403 104 -101 -207 78 10 427 -358 -150 -240 465 -352 -375 425 258 -342 443 319 341 -62 -260 -22 208 174 128 265 435 -429 213 272 167 311 -123 -82 -44 -193 428 -117 -51 -222 495 -86 279 -28 191 37 483 135 208 324 -427 448 -198 -366 475 -218 399 -238 206 464 -113 -127 -373 -384 292 83 -77 -428 -182 -276 -149 -186 491 -18 -362 182 -129 -379 -183 -421 -203 243 -473 452 377 -146 -266 -372 469 440 -56 -144 166 72 -28 -42 7 -253 30 326 -29 -267 -8 314 -433 130 -3 -62 -396 -334 17 -99 -239 397 -147 -362 -249 440 118 72 -268 -85 428 398 487 -100 208 346 499 91 172 -29 -324 16 -215 -256 -1 -366 182 103 -199 -300 -496 62 -403 210 -447 200 150 -477 -227 382 -62 53 281 277 -194 -11 476 -195 -68 148 -372 109 17 266 353 -484 400 35 119 53 -413 475 -32 36 185 373 89 -165 -252 -286 70 187 267 203 316 -75 44 292 231 477 293 359 86 162 477 -61 178 378 -174 149 283 266 124 -249 154 162 -24 -257 -151 225 309 419 264 77 122 432 -498 -333 225 -415 -356 370 -203 -418 32 -226 373 62 4 -301 -289 -212 -183 -313 391 472 349 -132 67 -301 -55 -123 -30 -291 -194 -407 -359 -192 112 -282 
500 180746
256 -60 -457 -310 -176 -183 -437 386 322 114 -51 -38 432 137 -147 -244 -14 -427 323 -463 18 -300 8 -421 6 453 -279 -333 65 291 -235 173 -268 -192 363 56 477 426 295 -349 40 -256 113 -28 381 -181 228 220 244 404 257 -237 -396 117 194 463 70 415 -18 -365 -441 247 308 143 -93 171 51 -116 449 -154 36 -10 91 -351 462 324 320 -457 44 65 447 154 -320 -97 123 374 -134 -306 -358 -300 181 -299 -201 342 344 206 365 -105 -409 315 242 127 -195 185 -372 119 9 -199 162 -94 366 461 -440 -454 -135 183 420 -417 -271 -438 -216 411 115 83 -395 311 142 470 -293 233 -363 -199 212 294 -162 340 -86 347 -359 -72 -247 -141 -110 165 -95 -393 -151 178 -310 430 -408 326 -307 -440 262 -202 371 -96 121 -422 -11 -390 231 -299 405 -431 -107 -181 -231 35 -401 22 -254 -11 40 -496 448 389 -466 -361 -329 -374 -183 -135 -314 -69 163 -90 335 136 340 176 247 72 377 152 493 -229 -177 -238 -342 -78 -363 -96 -236 177 -92 -288 418 -58 203 -411 421 -127 306 -41 304 -30 369 492 -42 210 168 -295 134 46 -291 479 -331 32 94 -173 -193 231 -417 71 260 344 -365 -470 138 339 -29 59 64 -222 -129 -132 -400 92 212 58 154 -119 -384 140 -221 177 120 300 -290 -286 -373 369 -203 -290 -60 -91 -94 427 -61 -455 -382 -238 456 182 40 179 -97 492 -228 -385 403 -222 348 -481 419 -21 -452 391 -221 -242 -395 -94 -21 254 469 419 -337 227 -301 454 272 -183 -284 229 -500 -391 -92 -97 -399 -468 18 -496 -189 219 -125 82 -302 -76 -27 -22 -466 430 -264 14 -464 57 285 -301 -215 484 5 -443 -198 -427 -362 -198 -318 399 -443 136 -217 427 140 94 146 -132 28 345 292 -499 -177 -322 -217 411 192 -181 -31 330 370 254 -334 -125 -337 -32 -51 -198 -378 483 53 -321 119 336 -393 112 283 -395 480 -189 450 -376 165 125 -198 -200 37 -153 120 -494 29 -10 260 195 -282 423 -336 -481 77 138 -498 -370 -330 474 -181 129 86 -398 234 418 265 185 -458 -70 -190 196 231 347 -105 203 -295 424 45 465 120 263 -259 284 282 -330 274 -363 153 444 -389 472 73 -451 -74 -192 467 -309 -155 -139 474 155 -443 -443 355 305 260 -440 229 -195 -122 -151 -79 471 -15 -297 -359 -240 192 294 -296 303 -382 278 -296 44 -62 -329 236 283 384 210 290 -206 267 497 -401 379 -90 328 -316 288 -322 105 259 163 -339 -248 423 353 -101 -20 -491 17 -242 -287 
500 -98938
-452 237 -350 -317 121 -288 473 415 479 -177 -486 -290 233 -305 -106 373 -127 -148 484 -112 13 -264 164 218 135 -356 227 5 254 -207 419 302 -470 -431 485 -497 -219 -190 -229 112 133 137 322 -134 332 68 91 -295 -228 75 446 285 -336 110 356 -349 254 435 156 8 -272 75 -190 110 496 147 466 129 457 -263 93 443 374 267 161 207 -164 -247 -236 108 328 -290 246 492 -328 454 -356 426 389 -348 286 -382 -420 -52 -420 -424 -405 -454 -442 -95 -217 151 -152 10 -81 -139 -283 -393 114 333 -433 295 -104 313 -361 68 267 135 347 157 288 133 -373 -280 -418 59 -204 29 106 -146 434 -259 -142 134 251 277 496 -180 384 462 -346 451 257 402 -383 -251 -30 -264 384 169 -255 -476 -197 224 96 237 -216 393 118 390 99 405 -17 -191 39 -265 -414 387 407 -178 -298 -87 126 -189 -185 243 60 -362 -169 297 307 77 -327 -38 153 270 -301 437 -485 318 -321 466 -425 163 -224 114 -250 214 354 -343 -463 -444 423 163 -133 -262 258 -220 -124 -411 -71 36 -482 102 498 172 224 50 461 239 220 -359 -442 295 -344 -166 -91 258 -452 115 -233 -63 -477 -310 -400 243 -71 358 -477 -343 448 304 193 -182 -94 192 490 -17 94 452 -278 -186 -55 132 -391 453 466 370 211 -133 -14 478 304 9 -331 -243 104 450 -385 127 -393 -85 283 -347 234 -310 -303 -424 173 291 -120 -253 -395 -323 -268 -286 -370 -302 -416 341 417 422 320 -426 284 -159 -169 -112 143 298 -132 -398 -286 -349 -245 -200 193 304 228 218 95 -391 466 -448 -214 -302 118 -83 248 -445 110 -482 477 282 -408 -387 -25 275 2 -382 73 370 -279 -361 -127 -24 -61 -433 -219 -332 285 -272 129 -397 133 415 153 -249 184 254 -194 -205 272 136 -71 -284 249 405 -157 -249 -477 416 -27 -256 -444 347 -427 347 -234 206 -133 -449 -214 496 -345 419 -236 -340 -329 448 414 -23 95 -462 -387 -475 106 -285 430 449 -34 -195 -282 -208 50 126 -9 475 -27 257 181 341 308 467 -311 -185 387 -47 -24 -90 254 -110 -261 201 281 205 -422 -113 420 -140 -163 238 166 55 -470 68 -319 21 43 -494 -222 76 347 -61 43 -111 254 282 -306 -270 44 448 473 284 -498 254 -11 -420 -359 -91 -59 330 -353 -393 -115 -322 175 418 -449 -282 425 -170 294 -376 -379 189 13 375 324 208 -42 -132 -492 283 -348 -490 389 141 -409 -118 -450 384 -287 50 491 450 228 -482 369 -369 -264 294 313 -118 418 434 -77 284 310 -253 
500 -35156
120 468 352 -97 -380 -285 -208 -386 158 174 16 42 -261 -434 385 -310 -354 403 -441 -222 -9 -295 -409 -275 -25 -474 148 -241 -312 -252 -397 308 -284 -192 63 188 375 355 302 33 381 -181 -73 -379 237 -188 -189 384 67 222 -338 -442 427 -247 -217 -98 131 -217 -486 319 31 -31 479 -401 -371 42 288 -496 249 442 389 483 113 -184 -396 -149 480 267 87 47 341 249 457 268 -146 -260 22 486 376 36 -343 -93 -142 -363 359 -13 -469 499 -8 -367 441 -119 -384 407 -450 72 110 -118 191 -303 -218 32 446 -261 -200 152 332 -326 138 208 -437 296 -33 -79 -215 -174 408 168 -175 -100 301 119 -366 -231 26 36 341 -12 -230 32 185 52 -436 483 -356 216 135 476 -257 126 184 -194 274 -349 227 -89 330 487 -421 -345 240 -267 274 -274 2 152 262 -156 -8 -116 -272 -323 437 -207 12 -419 -139 0 -443 104 -22 -407 -238 -248 96 -159 -485 -74 -171 -406 434 -431 179 208 147 -318 213 -91 378 -295 -355 -394 235 -418 251 99 -337 -35 -401 72 -431 429 -483 -168 33 466 173 48 -108 -146 495 178 275 174 239 -78 -292 -48 -317 -414 -491 -171 45 244 -89 148 -156 427 -387 295 351 -317 -423 -279 367 110 -313 392 -341 431 99 6 110 -126 -320 -151 297 -111 -347 -168 327 -338 161 -128 -241 -75 -479 103 204 -366 250 407 169 -321 -372 388 290 167 -367 -199 99 232 307 61 -42 487 410 107 228 -437 440 56 77 453 -220 336 -122 -199 291 -66 288 394 -158 309 -427 322 198 363 -10 -317 -336 -59 415 471 -498 -127 -189 -236 481 -461 179 -227 447 -244 -274 -272 -407 457 381 -264 243 169 -370 -63 331 56 -240 29 -81 -398 -436 436 43 479 -241 45 204 70 -339 -463 462 340 -190 409 -404 389 -363 41 -302 -129 -370 441 -460 -240 -121 371 316 139 252 88 -407 316 376 136 -353 -365 33 -296 -442 194 93 -128 34 404 -219 482 -355 271 24 -157 -358 154 -216 34 -234 163 258 -65 154 362 -125 247 -469 -249 235 30 238 -232 234 -352 -186 -172 20 -300 84 -346 -317 -419 425 -441 -76 -81 -435 -440 -195 183 -424 -437 -382 230 -222 345 330 -191 -52 65 339 39 -314 426 187 0 -246 -292 -447 -310 214 -264 -229 139 147 -453 -90 212 -393 -433 -105 -317 -369 366 -234 261 211 -404 -430 12 -487 -239 -449 51 -313 238 -96 -59 -202 -43 483 -136 193 -246 -145 -160 -199 265 -96 -91 333 299 444 -184 -483 -290 -423 -419 158 -1 93 -476 260 496 
500 219575
448 -414 479 -111 -115 -64 -127 -399 -19 479 -43 321 -219 -278 77 -458 -93 229 486 223 246 49 -348 327 -293 151 272 -269 -88 268 -341 -140 355 -362 -399 92 75 326 193 -444 -342 -350 -270 -61 225 307 333 -368 36 -329 -292 135 220 212 -38 -220 -136 87 363 -372 -145 -478 340 -290 -339 -207 -346 -412 -28 348 -4 130 498 226 -79 223 386 -246 -292 274 -75 -84 261 -2 -372 76 278 344 -337 493 -176 18 -484 -336 -419 29 310 -265 117 -218 -417 -35 412 -66 -308 185 9 -70 -209 217 56 216 -15 -182 66 466 394 196 -338 -91 190 -13 427 206 151 -140 87 -39 96 56 -405 -469 -479 -493 317 65 -456 -173 -153 -165 -104 404 -96 -266 74 470 -300 320 167 -138 -271 209 -299 8 267 353 369 -146 166 -35 410 262 348 431 -379 166 -3 18 493 344 353 241 -400 -243 475 -474 -272 27 346 -253 390 -73 456 -57 436 223 -204 305 -423 -185 122 -161 -71 -30 122 402 -364 -29 -80 -19 168 -226 -277 268 383 -302 -353 -37 226 493 210 468 -79 -482 -89 -143 241 -440 14 -330 227 -364 361 8 106 -16 -90 -405 455 -317 76 123 309 -349 244 192 -150 391 -344 -72 -264 -282 248 -491 89 11 218 -170 71 -268 353 150 -132 -434 -342 -173 50 421 274 358 -396 -150 333 -235 354 429 457 -296 -328 -35 484 -91 -464 232 -82 125 95 -363 -193 19 221 12 -331 -58 79 180 269 -371 101 395 -161 -443 -255 -327 -178 -49 -398 131 155 127 -51 491 -464 -15 -425 306 -38 171 -205 -379 -310 -483 134 211 -41 -287 -109 80 194 492 -25 -466 -99 -428 59 -425 24 -487 207 31 140 8 375 -472 -155 450 335 307 473 -18 428 -337 -1 62 375 -190 -373 -234 -258 -326 111 217 -292 -136 -358 -381 -60 -482 -16 -1 49 -23 -493 -76 5 -148 227 192 11 200 -325 -61 364 -474 -146 -409 189 333 209 431 -493 172 -499 67 -463 -357 186 -23 -339 23 -172 -438 -500 -165 339 5 187 66 50 -302 -234 -423 489 -18 -397 195 73 -356 29 135 -424 36 -193 -423 -396 196 72 -358 25 233 165 353 -353 -483 -312 486 375 -273 52 -75 277 -329 2 -381 153 457 314 79 -398 195 -286 30 -268 373 107 188 70 -321 -318 -405 -236 -300 -199 -89 -283 -11 250 -408 -431 -198 369 346 -27 -129 465 -21 181 -368 -90 -365 327 124 165 -441 349 124 -401 -81 -345 134 367 -81 334 -332 -318 -97 -491 432 -4 -422 87 -135 425 -88 -411 -258 -257 -378 -126 153 
500 216257
-298 -223 -226 113 -373 398 -435 398 -447 199 265 324 385 -215 -142 -212 -205 143 284 -127 -270 2 150 494 443 393 -262 -435 -233 -257 174 321 21 -200 -65 -500 -302 352 -250 103 51 -132 279 -64 153 -363 224 448 280 361 -178 -138 215 -176 209 -490 -283 -201 -73 337 42 -399 158 -85 -99 -55 267 451 -203 -482 54 -300 -114 185 136 -109 322 213 192 455 426 -134 169 141 190 -122 151 408 177 -70 245 -280 31 255 487 -216 -299 255 87 350 273 -7 -449 11 -322 -461 -246 -500 252 446 -193 -470 -188 477 -477 -497 -145 -474 411 -115 -44 -492 457 487 263 444 123 -184 199 -290 -333 324 203 -282 187 -267 109 442 -266 -138 388 393 244 53 370 268 56 78 294 319 -185 -249 -321 -228 90 -206 68 -286 111 120 276 -370 296 332 200 -16 -435 -191 278 151 171 -482 45 268 71 267 36 -21 -155 -170 -202 -488 433 329 -216 24 -376 -295 -410 -413 325 -282 -283 121 -450 417 -43 -32 -274 -265 -381 -250 -246 -484 -482 -323 284 54 157 -19 -264 -193 -6 -330 -363 -370 194 113 -165 -364 200 12 -146 417 -14 -95 334 -205 373 -440 31 -156 162 -363 -139 180 -186 -355 86 471 478 -325 -221 -28 -155 268 103 -109 -119 -210 27 -419 303 -267 -150 141 138 -316 436 363 96 319 60 -241 456 421 -209 -229 -82 -270 94 -104 -95 -275 -279 -398 -155 324 -7 226 -386 -128 -341 -231 105 9 410 96 -455 -301 311 142 -482 -129 -247 475 144 396 98 -86 -22 -308 -337 383 270 -116 485 -385 -292 330 342 174 -298 -147 -204 -340 -285 -294 256 -240 257 67 -246 276 439 359 -397 435 -392 201 -150 86 -255 13 322 -485 397 -341 -17 -43 490 177 -369 44 -470 279 204 -255 486 312 358 -405 -120 -388 371 171 -176 -174 106 -68 -121 -192 -130 477 321 -308 -156 -430 204 327 27 194 4 -489 90 387 290 -205 -368 -372 -393 -158 -276 339 307 -53 362 -369 274 -31 415 5 -371 -215 -18 451 330 327 -127 34 154 401 80 -341 -236 -330 -454 -446 317 30 35 -223 -275 -389 -384 -468 58 479 15 184 300 282 -310 429 -81 -476 -268 -251 351 106 -365 6 359 215 -483 -377 -262 -85 29 -445 -203 -84 184 22 27 301 406 -414 -368 273 -378 432 55 164 -287 475 189 298 76 40 256 212 -102 115 279 -85 -410 -131 -318 471 277 -20 -112 -39 -146 415 114 113 -147 246 -114 328 -470 442 -8 96 -231 -319 -106 345 -426 2 409 -28 -31 -459 -260 59 -90 274 
500 158882
-313 106 -230 149 -187 38 -237 426 -257 -138 164 -429 -108 458 64 488 -273 -403 234 -427 -329 -264 -166 496 -443 -125 88 468 286 -138 351 -175 -179 -27 -174 134 -489 442 412 -245 -196 76 -174 -452 35 -258 -463 262 -160 -377 -313 363 212 22 359 269 -251 299 238 -465 -338 441 -139 335 -86 187 -31 278 481 -119 -115 -215 -191 -437 -166 344 -194 223 459 498 -302 -354 361 410 -480 -427 -468 -230 372 -378 157 -114 -437 -130 -279 329 -90 -458 107 -109 -77 -156 177 233 -92 -489 429 -434 -414 388 -436 -216 -113 -223 47 -93 -150 79 -471 -277 201 187 461 264 -443 -317 93 -33 -275 53 359 -499 397 388 -414 157 -249 -485 223 -163 -244 -361 473 143 -83 20 -98 267 -401 -68 -158 -348 -29 -196 -232 380 -13 -286 200 -436 267 411 -83 -484 299 3 174 -450 19 249 239 -373 389 -436 270 -194 85 -328 -427 -464 456 268 -311 -73 -428 309 160 411 -477 -140 475 -358 -229 -107 11 -78 -252 -315 324 -381 -214 63 -254 -325 479 -484 -19 64 41 407 453 -3 175 -6 -223 247 -197 437 158 179 149 485 -327 -228 230 184 194 479 -279 18 -402 8 -67 -155 183 -88 213 17 -171 -246 -76 -218 252 -401 276 -119 346 -421 170 356 -390 -181 -307 -216 443 424 468 137 403 42 7 -147 -450 -208 -450 -415 -444 264 102 -115 -482 -474 -481 -378 -23 295 3 175 -273 25 31 -163 -304 -423 473 -361 -499 -206 -372 -244 336 487 109 -262 -221 -488 175 -164 276 130 -427 294 156 -407 -231 -14 388 -376 -339 -33 150 -455 -343 346 -378 -370 -162 -377 -76 318 231 112 306 192 350 -415 204 378 273 -20 8 347 127 16 -208 396 -498 -320 -480 -336 -500 -478 -291 -343 369 -169 -213 59 306 -436 -123 37 176 -317 -271 379 121 434 257 -106 266 117 93 -107 -367 385 -211 -364 -82 162 152 -82 184 -287 75 405 396 -286 -36 202 130 -306 91 -193 -123 320 38 -150 -394 147 245 -275 -236 -162 118 249 76 -240 385 494 422 389 -88 106 -398 -161 12 498 405 -172 200 36 22 143 -305 -248 316 -415 102 422 232 -153 -1 -152 -462 -382 -403 466 -122 -165 460 -200 -276 -128 406 179 -437 270 177 468 -401 230 356 121 -127 -449 373 -311 -364 -24 -36 -280 -325 463 68 65 -67 -482 31 311 -147 -9 -37 429 -285 222 108 130 -8 138 451 91 220 307 65 -55 -289 -62 487 199 -234 303 -228 294 266 -308 211 52 -290 -257 215 415 -414 -321 -303 154 401 -343 136 245 
500 -239205
87 -163 367 247 -246 -187 -190 192 -200 -139 -189 -397 133 -395 -279 -322 -332 273 -112 -89 -11 156 350 -480 -295 356 421 -138 344 166 10 284 -145 -123 31 109 190 -307 154 -158 406 -183 -55 40 274 -481 218 442 292 -42 206 133 -34 408 153 171 264 426 -114 108 445 396 244 152 125 127 -238 168 320 -232 -490 227 85 308 119 -141 327 189 -199 119 -1 -141 105 -182 -233 -242 -159 -469 -315 227 492 -370 475 236 282 101 216 -104 121 36 -484 131 115 453 -61 86 -188 118 -225 466 90 -373 -175 -305 -55 93 -47 286 476 490 366 -180 -28 341 -91 -245 294 -375 -497 -85 -487 -480 -101 481 -175 -310 67 138 309 -305 456 -101 174 133 446 -381 78 -101 -243 407 -110 -25 -273 362 -183 488 -31 -37 -35 325 -269 331 -303 482 312 22 24 231 12 -167 426 320 -416 100 -46 -470 -429 -468 282 -171 -209 24 304 371 238 -27 359 60 -211 177 -115 372 8 82 354 -180 -44 -122 -449 469 -436 -170 289 0 -218 95 383 -146 480 165 -465 -229 41 -309 -358 -221 -483 -146 339 158 -469 -276 30 39 -342 236 -289 467 -34 114 436 30 444 -423 -117 -273 173 -234 433 153 -217 -180 -224 324 11 271 -397 28 -375 -205 -314 8 371 -432 -453 -118 304 110 -151 -229 -276 137 -347 21 214 36 -400 239 -346 -467 244 -63 -147 -479 113 -284 292 -431 -403 -231 -136 135 277 -265 204 -324 117 8 286 -182 -369 -138 455 137 383 170 -327 -165 261 180 220 6 -31 -427 379 -417 142 -477 -348 239 -208 16 -126 -79 -397 -70 97 -427 -209 235 -109 -78 -51 199 -441 185 -131 85 -480 -18 -235 241 488 234 -334 -281 -331 -340 -258 -179 399 34 -311 126 -193 145 -444 404 218 -153 491 -39 -378 -59 -340 33 -374 381 -382 -2 -136 235 -261 204 322 258 424 491 418 -334 -187 -330 -447 354 296 212 -1 -296 117 69 52 460 -117 -474 253 43 59 379 277 178 230 -7 265 -179 -303 87 -421 -27 79 -150 140 -256 20 -455 -402 168 257 450 372 226 19 276 -461 402 302 292 298 -138 -476 75 -108 254 -432 -343 -425 117 245 7 -409 176 357 -417 420 229 -372 -130 -103 -263 -180 -231 -184 340 -454 -145 -406 348 -1 392 62 23 -33 454 -223 387 -36 -147 5 -291 -288 96 -263 -431 179 -343 298 -341 27 -305 248 -300 316 -84 -460 362 -377 -14 -437 123 -121 125 -354 -302 432 276 -62 -104 -19 443 457 193 391 -306 262 422 203 412 -419 82 -41 -319 
500 101782
-225 98 -326 -10 -427 161 405 196 -108 382 -305 90 314 471 -472 -290 -48 -177 19 -355 -286 213 -241 -364 268 -329 69 -149 130 -397 -367 -243 -447 160 247 -374 -179 -496 -325 213 387 -130 155 201 193 184 -236 -355 -141 -365 142 74 349 -247 -438 469 -76 484 -180 -94 -413 306 -485 -8 -34 -385 118 287 -381 -355 352 -494 15 7 -440 -292 -309 176 205 51 311 199 -23 -488 452 391 482 228 375 154 -366 -186 -40 2 -342 426 117 277 213 236 -226 65 -405 290 -427 7 -150 116 -317 -92 19 -154 -393 -152 211 -440 -260 -455 -360 -33 199 127 -366 -340 -371 -208 -414 246 -79 -348 334 196 217 429 338 -358 -64 -312 -389 471 448 130 170 -444 -169 -119 -32 71 -74 108 -110 -375 -265 24 137 -136 -331 224 -38 442 376 -203 490 445 -274 328 -60 -485 -131 51 486 -183 33 -492 -127 364 -111 -307 287 315 302 30 -207 -463 -94 430 254 75 -494 -284 370 382 13 360 -320 -408 189 472 -393 -90 23 -55 227 -92 -46 -47 125 -305 146 -88 -137 -52 -206 -492 338 201 438 444 -224 445 -340 -2 -321 -474 -141 211 -30 -452 183 429 -42 58 226 37 467 32 490 92 228 489 -144 -409 289 151 -401 479 -148 -111 423 480 186 -64 -21 -282 314 -310 429 -216 90 113 -287 -452 -329 439 437 -10 -28 428 -66 -300 -231 -357 143 -442 294 -406 -462 -2 -17 461 330 22 249 161 240 63 351 169 347 -59 134 -88 341 -342 -148 279 148 -324 -441 -417 -124 -172 -274 -129 238 -128 317 276 -278 152 -410 -448 -326 339 -434 414 -97 -231 436 102 211 70 -485 404 228 219 -465 229 395 -406 312 123 274 -110 -154 13 -386 -337 141 -164 315 231 240 490 71 -342 -244 -174 -72 -308 -220 -361 -385 147 -105 343 -134 -69 72 113 377 236 88 152 -22 434 17 92 -403 10 428 -235 -258 -479 -245 -335 -321 363 343 107 -444 -377 -402 -329 271 -6 -134 -11 -223 -209 -45 -346 -473 -457 306 6 478 175 -402 -73 -314 -121 192 280 252 447 445 -217 -337 288 -257 71 -237 -159 -406 -466 -313 -40 376 -36 251 -169 119 -369 -126 277 137 204 305 -413 -368 491 318 324 271 70 -376 -432 354 -213 356 -403 -290 -381 290 -196 6 -170 264 234 294 368 -83 265 -149 -357 395 488 348 200 -73 480 -457 -254 -344 314 -332 -220 -266 -478 419 -58 -29 -371 413 114 -67 -81 444 50 5 238 270 422 356 -379 -434 251 -39 -234 -49 -260 98 346 -14 254 -488 -493 -113 -254 
