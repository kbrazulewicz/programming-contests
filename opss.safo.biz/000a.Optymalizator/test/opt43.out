NIE
TAK
NIE
NIE
TAK
NIE
NIE
NIE
NIE
NIE
